cmd/test-server/networking.go

```diff
@@ -37,20 +37,20 @@ type Results struct {
 	AvgIcmpStat float64
 }
 
-// IcmpPinger sends ICMP pings and returns statistics
-func IcmpPinger(ip string) PingMsmt {
+// icmpPinger sends ICMP pings and returns statistics
+func icmpPinger(ip string) (PingMsmt, error) {
```

Comment:
Nitpick: I'd return a pointer to the struct, so we don't have to copy around several fields.
```suggestion
func icmpPinger(ip string) (*PingMsmt, error) {
```

Reply:
Done in ef1dab2691d68cf84fe8a3c687ab9e19fd6a6c40